examples/src/main/scala/org/apache/spark/examples/streaming/ZeroMQWordCount.scala

```diff
@@ -90,7 +89,9 @@ object ZeroMQWordCount {
     def bytesToStringIterator(x: Seq[ByteString]): Iterator[String] = x.map(_.utf8String).iterator
 
     // For this stream, a zeroMQ publisher should be running.
-    val lines = ZeroMQUtils.createStream(ssc, url, Subscribe(topic), bytesToStringIterator _)
+    val lines = ZeroMQUtils.createStream[String](
```

Comment:
Need to use `createStream[String]` to make the compiler happy. There are two `createStream` with default parameters and looks the compiler is confused. Maybe we should use a new method name for the new `createStream` with `actorSystemCreator`.